apps/emqx_utils/src/emqx_utils.erl

```diff
@@ -289,8 +289,10 @@ do_check_oom([{Val, Max, Reason} | Rest]) ->
     end.
 
 tune_heap_size(#{enable := false}) ->
-    ok;
+    ignore;
 %% If the max_heap_size is set to zero, the limit is disabled.
+tune_heap_size(#{max_heap_size := 0}) ->
+    ignore;
```

Comment:
There's currently no mention that setting 0 is equivalent to disabling. I guess this needs to be mentioned in `emqx_schema.hocon`?

Reply:
True, there's only this comment and the [originating issue](https://github.com/emqx/emqx/issues/13417).

Reply:
:checkered_flag: